### ISSUE TITLE
Provide keybindings to copy entries to the clipboard

### ISSUE THREAD

**Issue by blueyed** [davatorium/rofi#378]
I would be useful if you could copy a selected entry as through a generic keybinding.

`Ctrl-c` to copy it to the clipboard might be enough, not sure about (and what to use for) the primary selection (`<Shift-Delete>`?!)

Ref: https://github.com/DaveDavenport/rofi/issues/174#issuecomment-207464443

**Comment by dragon788** [comment]
Personally I'd like to see this as <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>C</kbd> if only because that's how I copy from my terminal (most VTE based like gnome-terminal/etc) and odds are if I'm copying a command to further edit I'm going to be pasting it into a terminal with the corresponding <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>V</kbd>.

**Comment by svenstaro** [comment]
This would be pretty cool. @DaveDavenport your thoughts on this? If you don't think it'd be good to have this in rofi, I'd have to implement it just for my plugin.

**Comment by DaveDavenport** [comment]
Patch is welcome.  Never got around implementing it because I was unsure if possible as normally (if I understand correctly) there needs to be an 'owner' of the clipboard content, and if rofi is closed.....

**Comment by dragon788** [comment]
If you could bind an action to a keypress within rofi you could have it pipe out to xsel or xclip or pbcopy and not have to manage the clipboard within rofi itself. That would be more in keeping with the Unix philosophy.

**Comment by anarcat** [comment]
my experience (at least from memory) of clipboard ownership is that it varies from one application to the next. in general, my "feeling" is that the primary selection (text that is just selected with the left mouse button and pasted with the middle mouse button) goes away when the process dies, but that CLIPBOARD (<kbd>control-c</kbd>) selections survive processes. `xclip` treats both equally and forks in the background to service the selection requests, exiting when it detect something else is responsible. *but* the [specification](https://www.freedesktop.org/wiki/ClipboardManager/) actually says that:

> If a client needs to exit while owning the CLIPBOARD selection, it should request the clipboard manager to take over the ownership of the clipboard, using the SAVE_TARGETS mechanism. If there is no clipboard manager, or if the SAVE_TARGETS conversion fails, the application should simply exit. 

In other words, there is a way to save clipboard contents outside of the application when we exit, so it should be possible to implement clipboard support without reverting to hacks involving xclip.

**Comment by robole** [comment]
This would be a "feature" for me. I regularly used my previous launcher as a quick and dirty pastebin to strip rich formatting from text I grab from websites before adding it to an email. I find it hard to shake this habit! It would be great if you get around to it. 😀🙏🏻 Thanks for a great program and keep up the good work
